hyperparameter_search.py

```diff
@@ -354,24 +358,28 @@ def search_hyperparameters(model, no_of_epochs, no_of_trials, dataset, data_csv,
             transforms.Resize((299, 299)) if MODEL_NAME == "inception_v3" else transforms.Resize((256, 256)),
             transforms.ToTensor(),
             transforms.Normalize(mean=mean, std=std),
-            # GaussianNoise(0., 0.1),
         ])
     else:
         data_transform = transforms.Compose([
             transforms.ToPILImage(),
             transforms.Pad(50),
-            transforms.Resize((299, 299)) if model == "inception_v3" else transforms.Resize((256, 256)),
+            transforms.Resize((299, 299)) if MODEL_NAME == "inception_v3" else transforms.Resize((256, 256)),
```

Comment:
I would add get_image_size function to return the supported image size for the model introduced in the commit 206a76fb177a57e5fbbdbe69c021544a27208004 

```suggestion
            transforms.Resize(image_size),
```